settings.yml

```diff
@@ -2,7 +2,7 @@ svg_icons:
   default: "moon|sun"
   type: "list"
   list_type: "compact"
-add_color_scheme_toggle_to_header:
+remove_color_scheme_toggle_from_header:
   default: false
```

Comment:
Could we just change the default to `true`? Otherwise all those who had it enabled will have it disabled and vice versa 😅

Reply:
sure!